elements/lisk-p2p/src/validation.ts

```diff
@@ -39,6 +43,7 @@ import { constructPeerIdFromPeerInfo } from './utils';
 
 const IPV4_NUMBER = 4;
 const IPV6_NUMBER = 6;
+export const IPADDRESS_FIELD_SUPPORT_VERSION = '2.3.0';
```

Comment:
`IP_ADDRESS_FIELD_SUPPORT_VERSION` is better.

Reply:
Is this SDK version or Core version?
I think this version doesnt relates to this library, so Prefer not to use hardcoded version , too.